src/main/java/com/google/sps/servlets/MentorApprovalServlet.java

```diff
@@ -77,6 +78,29 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     // Update database tables related to mentor approval.
     addApproval(mentorId, approverId);
     addEvidence(isApproved, mentorId);
+
+    // If mentor review is complete, send them a notification.
+    String notificationType = "";
+    if (Utility.getReviewStatus("is_approved", mentorId) == Utility.MENTOR_APPROVED) {
+      // If mentor is approved, send notification of type 'approved'.
+      notificationType = "approved";
+    } else if (Utility.getReviewStatus("is_rejected", mentorId) == Utility.MENTOR_REJECTED) {
+      // If mentor is rejected, send notification of type 'rejected'.
+      notificationType = "rejected";
+    }
+
+    // Post to notification servlet.
+    if (Utility.getReviewStatus("is_approved", mentorId) == Utility.MENTOR_APPROVED) {
```

Comment:
If the notification is "rejected", the notification will never send here. I think you can just check if the notificationType is an empty string (meaning it was either accepted or rejected) instead of checking the reviewStatus again.

Reply:
Rethinking all this, I think there is no necessity of having the getReviewStatus() return a constant, so I made it so it returns the string directly. It receives either `approved`, `rejected`, or empty, and if it is not empty, it sends the notification. Do you agree with this change?